docs/development/angular/performance-cookbook/angular8-ivy.md

```diff
@@ -0,0 +1,135 @@
+# Angular 8+ (Ivy)
+
+## Table of contents
+
+* [Asynchronous components](#asynchronous-components)
+
+# Asynchronous components
```

Comment:
Use only one h1 header per page
It is allowed to build a more consistent sidebar menu
But don't forget to set `subMaxLevel` to `2`